config/mfDefs.gem

```diff
@@ -43,8 +43,8 @@ UserDynamicLinkFlags =
 IncludeDirs+=$(XDAQ_ROOT)/include
 
 ## Python
-PYTHON_VERSION ?= $(shell python -c "import distutils.sysconfig;print distutils.sysconfig.get_python_version()")
-PYTHON_INCLUDE_PREFIX ?= $(shell python -c "import distutils.sysconfig;print distutils.sysconfig.get_python_inc()")
+PYTHON_VERSION ?= $(shell python2.7 -c "import distutils.sysconfig;print distutils.sysconfig.get_python_version()")
+PYTHON_INCLUDE_PREFIX ?= $(shell python2.7 -c "import distutils.sysconfig;print distutils.sysconfig.get_python_inc()")
```

Comment:
As long as the setup script for `rh-python27` is run (simple enough to put in a dotfile), it will work the same at *all* teststands (and p5), which makes it much more appealing if it's done this way